src/value_converter.rs

```diff
@@ -744,6 +745,84 @@ pub fn py_sequence_into_postgres_array(
     }
 }
 
+/// Extract a value from a Python object, raising an error if missing or invalid
+///
+/// # Type Parameters
+/// - `T`: The type to which the attribute's value will be converted. This type must implement the `FromPyObject` trait
```

Comment:
It is specified in types

```suggestion
/// - `T`: The type to which the attribute's value will be converted.
```